R/utilities-scenario-configuration.R

```diff
@@ -84,41 +96,18 @@ readScenarioConfigurationFromExcel <- function(scenarioNames, projectConfigurati
   return(scenarioConfigurations)
 }
 
-#' Set an application protocol in a `Simulation`
+#' Set an application protocol in a `Simulation` from the excel file.
 #'
 #' @details Set the parameter values describing the application protocol
-#' defined in the scenario configuration. Either calling a function that is stored
-#' in the `applicationProotocolsEnum`, or from excel.
+#' defined in the scenario configuration.
 #'
 #' @param simulation A `Simulation` object that will be modified.
 #' @param scenarioConfiguration A `ScenarioConfiguration` object holding the
 #' name of the application protocol.
-#' @param applicationProtocolsEnum (Optional) A named list with functions that
-#' define setting the application. If `NULL` or no entry with the application
-#' protocol defined in the `scenarioConfiguiration`, the parameters to set
-#' are extracted from excel.
 #'
 #' @export
-setApplications <- function(simulation, scenarioConfiguration, applicationProtocolsEnum = NULL) {
```

Comment:
Only setting applications from excel. This will not break anything as `applicationProtocolsEnum` was never used.